meta/main.yml

```diff
@@ -8,9 +8,10 @@ galaxy_info:
   platforms:
   - name: OpenBSD
     versions:
+    - 6.2
```

Comment:
the issue #296 of `galaxy` has been resolved?

Reply:
fixed in d2d597f